linux-object/src/fs/file.rs

```diff
@@ -44,6 +44,8 @@ pub struct OpenOptions {
     pub append: bool,
     /// non block open
     pub nonblock: bool,
+    /// close on exec
+    pub fd_cloexec: bool,
```

Comment:
NOTE:
The `cloexec` flag can be set through `fcntl(fd, F_SETFD, FD_CLOEXEC)`.
So it should be put into the inner or as an `AtomicBool` in the future.